### ISSUE TITLE
Support a `.spellignore` file

### ISSUE THREAD

**Issue by silverwind** [golangci/misspell#23]
Misspell currently supports a `-i string` argument but I find it more flexible to have ignored words listed in a file. In the spirit of `.gitignore` and other similar files, I propose supporting a `.spellignore` file. Currently I have implemented this in a bash script:

```bash
#!/bin/bash
MISSPELL_PACKAGE="github.com/golangci/misspell/cmd/misspell@v0.6.0"

if [ -e .spellignore ]; then
  go run "$MISSPELL_PACKAGE" -i "$(xargs echo -n < .spellignore | tr ' ' ',')" "$@"
else
  go run "$MISSPELL_PACKAGE" "$@"
fi
```

**Comment by ldez** [comment]
I can see 2 problems:
- `.spellignore` name implies a kind of proximity with `.gitignore` syntax, and it's not something interesting in the context of misspell.
- as the list of words contains common mistakes, the ignore list should be very small unless you want to keep mistakes.

Your issue is more a proposal of a solution than a description of a problem.
Even if I understand the needs, the proposed solution is not something I want to implement.

I can see 2 other approaches:
- create a configuration file with all the options of misspell
- use the same approach as `-dict` by adding an option to load a file.

**Comment by silverwind** [comment]
> use the same approach as -dict by adding an option to load a file.

Yes, loading ignore words from a file instead of having to pass them ono by one on the command line via `-i` would be a improvement. You don't even have to name a file if you make the option like `--ignorefile file`, so the user chooses the name.

> as the list of words contains common mistakes, the ignore list should be very small unless you want to keep mistakes.

Yes, the amount of false positives of this module is low, but not zero and can never be zero. One common source of false positives is non-english content, for example german word `initiales`. This can happen  when quoting content written in non-english languages.
